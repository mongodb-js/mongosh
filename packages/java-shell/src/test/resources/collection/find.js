// before
db.coll.remove({});
db.coll.insertOne({key: "value", array: [1, 2, 3, {another_object: "  .$# "}]});
// command
db.coll.find();
// clear
db.coll.drop();
