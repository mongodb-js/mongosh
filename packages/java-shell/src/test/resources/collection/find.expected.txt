[ { "_id": <ObjectID>, "key": "value", "array": [ 1, 2, 3, { "another_object": "  .$# " } ] } ]